Training/Training/file.c

```diff
@@ -78,35 +85,41 @@ void SearchAndSort () {
    for (int i = 0; i < size; i++) {
       char index[] = "[i]";
       index[1] = i + 48;
```

Comment:
Here you can directly print the index outside itself. Creating a char array just to print the index makes it complex.